extensions/apiconnect/src/apiconnect.spec-enhancer.ts

```diff
@@ -23,9 +29,18 @@ export class ApiConnectSpecEnhancer implements OASEnhancer {
 
   constructor(
     @config({optional: false}) private options: ApiConnectSpecOptions,
+    @inject(CoreBindings.APPLICATION_METADATA, {optional: true})
+    private appMetadata: ApplicationMetadata = {
+      name: 'LoopBack Application',
```

Comment:
can we get the default value for x-ibm-name from the app's package.json

Reply:
That's what the `inject` about. The `LoopBack Application` is the default value if application package is not loaded.

Reply:
`CoreBindings.APPLICATION_METADATA` takes care of injecting the package json , is it ?